Bindings~/python/tests/integration/test_scene01.py

```diff
@@ -818,7 +819,8 @@ def test_input_field_events(self):
         assert input_field.get_component_property(
             "AltInputFieldRaisedEvents", "onSubmitInvoked", "Assembly-CSharp")
 
-    @pytest.mark.WebGLUnsupported
+    # @pytest.mark.WebGLUnsupported
```

Comment:
Don't remove the `@pytest.mark.WebGLUnsupported` and you can add a similar one for iOS.